### PR TITLE
Remove DOB and SSN from veteran_representatives

### PR DESCRIPTION
**Pull request by stevenjcumming**
## Summary

- Remove `dob` and `ssn` fields from `Veteran::Service::Representative` model
- Remove `has_kms_key` and `has_encrypted` from `Veteran::Service::Representative` 
- VERY minor refactoring to rename `rep` variables to `representative` for clarity 

Migration related code moved to different PR https://github.com/department-of-veterans-affairs/vets-api/pull/16377

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/80875
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/80806

## Testing done

- [ ] test suite passes

## Acceptance Criteria

- [ ] `dob` and `ssn` fields are removed `Veteran::Service::Representative` model 
- [ ] remove has_kms_key from model

### CODE REVIEW

File: db/schema.rb
Comment (jvcAdHoc) [inline db/schema.rb:1]: Are all of the `precision: nil` changes intentional?

**Comment by jvcAdHoc** [comment]
Overall this looks good to me. I don't know how safe we need to be with migrations + code changes in the same pr. Might be worth doing:
1. a pr to ignore the columns in the model + remove their use in the model methods
2. a migration pr
3. a ignore columns cleanup pr

File: db/schema.rb
Comment (stevenjcumming) [reply db/schema.rb:1]: Yes, I'm not sure how but they keep getting added back in. I should have mentioned that in the PR description.

File: modules/veteran/app/models/veteran/service/representative.rb [hunk 1]
```diff
@@ -9,8 +9,6 @@ class Representative < ApplicationRecord
       BASE_URL = 'https://www.va.gov/ogc/apps/accreditation/'
```
Comment (jvcAdHoc) [inline modules/veteran/app/models/veteran/service/representative.rb:10]: Something like this to adhere to strong_migrations recommended process

```suggestion
self.ignored_columns += %w[dob_ciphertext encrypted_kms_key ssn_ciphertext]
```

**Review by jvcAdHoc — APPROVED** [review]

**Comment by stevenjcumming** [comment]
@jvcAdHoc, you are right I should have made separate PRs. Thanks

**Comment by stevenjcumming** [comment]
@jvcAdHoc I've removed the `ignore_column`. It's causing some tests to break, but I think it will be ok

**Review by jvcAdHoc — APPROVED** [review]

**Pull request merged**